Makefile

```diff
@@ -17,14 +17,13 @@ build-content:
 
 build: clean build-content
 
-htmlproofer-setup:
-	gem install bundler
-	NOKOGIRI_USE_SYSTEM_LIBRARIES=true bundle install \
-	--path vendor/bundle
+link-checker-setup:
+	curl https://htmltest.wjdp.uk | bash
```

Comment:
kind of weird to be downloading executable script from a private domain like this.